Update links

rjc opened the pull request:
- _http_ `->` _https_ where appropriate
- _wiki.openra.net_ `->` _https://github.com/OpenRA/OpenRA/wiki_
- _bugs.openra.net_ `->` _https://github.com/OpenRA/OpenRA/issues_
- the IRC channel is now on Libera.Chat + make it into a URI

This is in order to be consistent (one link to GitHub wiki already
exists), reflect reality, as well as avoid unnecessary redirects.

abcdefg30 commented on README.md:
```diff
@@ -24,29 +24,29 @@ Check our [Playing the Game](https://github.com/OpenRA/OpenRA/wiki/Playing-the-g
-* Download a copy of the [OpenRA Mod SDK](https://github.com/OpenRA/OpenRAModSDK/) to start your own mod.
-* Check the [Modding Guide](http://wiki.openra.net/Modding-Guide) to create your own classic RTS.
+* Download a copy of the [OpenRA Mod SDK](https://github.com/OpenRA/OpenRAModSDK) to start your own mod.
+* Check the [Modding Guide](https://github.com/OpenRA/OpenRA/wiki/Modding-Guide to create your own classic RTS.
```
This accidentally drops a `)`:
```suggestion
* Check the [Modding Guide](https://github.com/OpenRA/OpenRA/wiki/Modding-Guide) to create your own classic RTS.
```

abcdefg30 reviewed (commented):
Lgtm otherwise.

rjc commented:
Argh! Sorry :^(

abcdefg30 commented:
> Argh! Sorry :^(

Don't worry. ^^
Can you squash the commits together, please?

Commit:
Update links

- _http_ `->` _https_ where appropriate
- _wiki.openra.net_ `->` _https://github.com/OpenRA/OpenRA/wiki_
- _bugs.openra.net_ `->` _https://github.com/OpenRA/OpenRA/issues_
- the IRC channel is now on Libera.Chat + make it into a URI

This is in order to be consistent (one link to GitHub wiki already
exists), reflect reality, as well as avoid unnecessary redirects.

rjc commented:
>  Can you squash the commits together, please?

Done.
